docs/creating-service-bindings/creating-service-binding.md

```diff
@@ -5,7 +5,120 @@ sidebar_position: 1
 # Creating Service Binding
 
 In this section, we consider that the service you want to connect to is already exposing binding metadata by either:
-- Beeing part of the "known" services
+- Being part of the "known" services
 - Exposing binding-metadatas by default
 If it's not the case, please refer to the following section(#)
 
+
+## Service Binding Resource
```

Comment:
Can you move this to: https://github.com/slemeur/service-binding-documentation/blob/main/docs/creating-service-bindings/service-binding-resource.md

Reply:
Okay. Yes.